pilot/pkg/kube/inject/webhook_test.go

```diff
@@ -1370,7 +1374,7 @@ func TestRunAndServe(t *testing.T) {
 			}
 
 			if !bytes.Equal(gotPatch.Bytes(), wantPatch.Bytes()) {
-				t.Fatalf("got bad patch: \n got %v \n want %v", gotPatch, wantPatch)
+				t.Fatalf("got bad patch: \n got %v \n want %v", gotPatch.String(), wantPatch.String())
```

Comment:
Is this necessary? I thought it calls `.String()` automatically?

Reply:
Oh, no this is not strictly necessary. Used it for debugging; it was spitting out byte arrays like
```
[25 05 155 ...]
```

and I wanted to look at the string representation. I can remove it again.

Reply:
No, if it helps, then leave it in. I just thought Fatalf already calls .String(), but apparently not?

Reply:
Yeah, definitely leave `.String()` in.